leasing/report/invoice/invoicing_review.py

```diff
@@ -458,108 +456,123 @@ def get_gaps_in_billing_periods_data(
         """
 
         query = """
-            SELECT li.identifier AS "lease_identifier",
-                l.id AS "lease_id",
-                l.start_date,
-                l.end_date,
-                r.id AS "rent_id",
-                r.start_date AS "rent_start_date",
-                r.end_date AS "rent_end_date",
-                i.id AS "invoice_id",
-                i.billing_period_start_date,
-                i.billing_period_end_date
-            FROM leasing_lease l
-            INNER JOIN leasing_leaseidentifier li
-                ON l.identifier_id = li.id
-            INNER JOIN leasing_rent r
-                ON l.id = r.lease_id
-            LEFT JOIN leasing_invoice i
-                ON l.id = i.lease_id
-            INNER JOIN leasing_invoicerow ir
-                ON i.id = ir.invoice_id
-            WHERE (l.end_date IS NULL OR l.end_date >= %(today)s)
-                AND (l.start_date IS NOT NULL AND l.start_date <= %(today)s)
-                AND l.service_unit_id = ANY(%(service_units)s)
-                AND l.deleted IS NULL
-                AND l.state NOT IN ('reservation', 'power_of_attorney')
-                AND r.deleted IS NULL
-                AND r."type" != 'free'
-                AND (r.start_date IS NULL OR r.start_date <= %(today)s)
-                AND (r.end_date IS NULL OR r.end_date >= %(today)s)
-                AND i.deleted IS NULL
-                AND i.deleted_by_cascade IS false
-                AND ir.deleted IS NULL
-                AND ir.receivable_type_id != ANY(%(excluded_receivable_type_ids)s)
-            ORDER BY li.identifier ASC;
+WITH invoices_sorted AS (
```

Comment:
The query is called `invoices_sorted`, but I don't see an ORDER BY besides the one inside the LEAD-OVER expression. Is that enough to sort the entire result?

Reply:
That is a good point, I don't recall where it went. I will add back ordering and see if it changes results

Reply:
@juho-kettunen-nc I have updated this and removed the `_sorted` part, there is no difference to sort the first query.